pkg/storaged/lvol-tabs.jsx

```diff
@@ -509,6 +509,9 @@ export const VDOPoolTab = ({ client, lvol }) => {
     const vdo_iface = client.vdo_vols[lvol.path];
     const vdo_pool_vol = client.lvols[vdo_iface.VDOPool];
 
+    if (!vdo_pool_vol)
+        return null;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18762-20230508-094606-33167c58-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)